pypesto/visualize/profiles.py

```diff
@@ -167,7 +179,11 @@ def profiles_lowlevel(fvals, ax=None, size=(18.5, 6.5), color=None,
                                            legend_text=tmp_legend)
 
             # labels
-            ax[counter].set_xlabel(f'Parameter {i_plot} value')
+            if x_labels is None:
+                ax[counter].set_xlabel(f'Parameter {i_plot} value')
```

Comment:
```suggestion
                ax[counter].set_xlabel(f'Parameter {i_plot}')
```

Reply:
do you think rerunning rosenbrock example and a short remark on custom x_labels will suffice?

Reply:
Maybe you can rerun rosenbrock, but no need to change anything there, and upload the picture here in the discussion of the PR.